components/rollup_node/consensus_layer.md

```diff
@@ -0,0 +1,27 @@
+# Consensus Layer
+
+The consensus layer is the core driver which guarantees that the canonical L1 state is accurately reflected in the EE. It does this by connecting to an L1 node, and using the pure function which maps L1 blocks to L2 block inputs as defined by [block generation][block-gen]. Once it knows the block inputs, it checks the EE to make sure that the blocks there all have the correct inputs.
+
+This is a functionality that is shared between all configurations of the rollup node, including the sequencer. Because block generation defines the L1 block -> L2 block input transformation as a pure function, it can be completely stateless and has no DB.
+
+## Basic Functionality
```

Comment:
I've confirmed with proto that this logical loop is fake news, we just can sync over geth p2p, attempt to get to the sequencer tip, and it will either succeed, or fail because it could note get backwards to the L1 confirmed state, the sequencer must be lying

Reply:
BUT, leaving that clarification to a future PR by @protolambda who understands much better.